src/App.jsx

```diff
@@ -1,45 +1,12 @@
 import React from 'react';
 import './App.scss';
-
-export const misha = {
-  name: 'Misha',
-  age: 37,
-  sex: 'm',
-  isMarried: true,
-  partnerName: 'Natasha',
-};
-
-export const olya = {
-  name: 'Olya',
-  sex: 'f',
-  isMarried: true,
-  partnerName: 'Maksym',
-};
-
-export const alex = {
-  name: 'Alex',
-  age: 25,
-  sex: 'm',
-  isMarried: false,
-};
+import { Person } from './components/Person/Person';
```

Comment:
If you follow my previous comment and will add index.js to component directory - this will simplify import here, it will become just: `import { Person } from './components/Person';` since index.js is always considered as a default directory for importing something from the folder.